bestla/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ if(BTLA_SYCL)
   file(GLOB sycl_headers ${PROJECT_NAME}/sycl/*.h ${PROJECT_NAME}/sycl/*.hpp)
   add_compile_definitions(BTLA_SYCL)
   list(APPEND sycl_libs IntelSYCL::SYCL_CXX)
+  add_compile_options(-march=native)
```

Comment:
will this change make bestla contain some instructions which customers' machines don't support?
e.g. we compile & release the package by spr/emr, but user use Core processor to use bestla.

Reply:
We won't use dpcpp to release binary packages. So it won't be a problem. Dpcpp is only for local compile&run.